bucket/tachidesk.json

```diff
@@ -0,0 +1,31 @@
+{
+    "description": "A rewrite of Tachiyomi for the Desktop",
```

Comment:
Completely nothing-saying and nonsense description. For end user absolutely unusable description

Reply:
That was based off their repo about section. So here goes the new one.